src/sounds.h

```diff
@@ -10,7 +10,17 @@
 class monster;
 class player;
 
+struct sound_event {
+    int volume;
+    std::string description;
+    bool ambient;
+    bool footstep;
+};
+
 namespace sounds {
+    // The sound events currently displayed to the player.
+    static std::unordered_map<point, sound_event> sound_markers;
```

Comment:
Please add the include for unordered_map to the header next time. It works currently because enums.h includes json.h which includes unordered_map, but that is just luck.